modin/pandas/groupby.py

```diff
@@ -459,8 +459,42 @@ def indices(self):
         return self._indices_cache
 
     @_inherit_docstrings(pandas.core.groupby.DataFrameGroupBy.pct_change)
-    def pct_change(self, *args, **kwargs):
-        return self._default_to_pandas(lambda df: df.pct_change(*args, **kwargs))
+    def pct_change(self, periods=1, fill_method="pad", limit=None, freq=None, axis=0):
+        from .dataframe import DataFrame
+
+        # Should check for API level errors
+        # Attempting to match pandas error behavior here
+        if not isinstance(periods, int):
+            raise TypeError(f"periods must be an int. got {type(periods)} instead")
+
+        if isinstance(self._df, Series):
+            if not is_numeric_dtype(self._df.dtypes):
+                raise TypeError(
+                    f"unsupported operand type for -: got {self._df.dtypes}"
+                )
+        elif isinstance(self._df, DataFrame) and axis == 0:
```

Comment:
How come we don't check if `axis==0` in https://github.com/ponder-org/modin-public/commit/2402cc17ca88e38066830a509d3e5a5bee02ab31? Is this because we only support `axis==0` there, so we need to add this check here?

Reply:
It's a bug in that commit. We don't test axis=1 in the ponder repo because we don't support it there.